docs/en/alab/project_creation.md

```diff
@@ -0,0 +1,81 @@
+
+<img class="image image--xl" src="/assets/images/annotation_lab/4.1.0/reserved.gif" style="width:100%; align:center; box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);"/>
+
+**Adding Team Members**
+
+When working in teams, projects can be shared with other team members.
+
+The user who creates a project is called a Project Owner. He/She has complete visibility and ownership of the project for its entire lifecycle. If the Project Owner is removed from the user database, then all his/her projects are transfered to a new project owner. The Project Owner can edit the project configuration, can import/export tasks, can create a project team that will work on his project and can access project analytics.
+When defining the project team, a project owner has access to three distinct roles: Annotator, Reviewer, and Manager. These are very useful for most of the workflows that our users follow.
+
+- An Annotator is able to see the tasks which have been assigned to him or her and can create annotations on the documents.
+- The Reviewer is able to see the work of the annotators and approve it or reject in case he finds issues that need to be solved.
+- The Manager is able to see the work of the Annotators and of the Reviewers and he can assign tasks to team members. This is useful for eliminating work overlap and for a better management of the work load.
+
+To add a user to your project team, navigate to the Project Setup page. On the Manage Project Team tab, start typing the name of a user in the available text box. This will populate a list of available users having the username start with the characters you typed. From the dropdown select the user you want to add to your team. Select a role for the user and click on the “Add to team” button.
+
+In the Project Team page users can add/remove/update the team members even in the case of a large number of members. The team members are displayed in a tabular view. Each member has a priority assigned to them for CONLL export which can be changed by dragging them across the list.
+![teamMembers](https://user-images.githubusercontent.com/46840490/193060010-3394cccd-93ab-40a8-8479-98155ea8b417.gif)
+
+**Project Configuration**
+
+The Project Configuration itself is a multi-step process. The wizard will guide users through each step and provide information and hints for each step.
+![projectConfiguration](https://user-images.githubusercontent.com/46840490/193033349-534cc2ab-2e5a-4caa-a050-0ee650949b21.gif)
+
+## Clone
+You can create a copy of a project, by using the Clone option. The option to clone the project is also listed in the kebab menu of each project. The cloned project is differentiated as it contains cloned suffix in its project name.
+
+<img class="image image--xl" src="/assets/images/annotation_lab/4.1.0/dashboardCloneImportExport.png" style="width:100%; align:center; box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);"/>
```

Comment:
You can remove the `align:center;` from the style attribute as it is incorrect property field. And instead of `box-shadow: ....` you can add the class `image__shadow` to the class attribute.